feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -0,0 +1,101 @@
+package com.iteneum.office.presentation.ui
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Call
+import androidx.compose.material.icons.outlined.Mail
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import com.iteneum.designsystem.components.LPBodyMedium
+import com.iteneum.designsystem.components.LPTitleLarge
+import com.iteneum.designsystem.components.LpOutlinedButton
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.office.R
+import com.iteneum.office.data.OfficeModel
+
+/**
+ * function: OfficeView  will handle:
+ * Exposing to the UI the Office information,
+ * working as container for OfficeUI function.
+ * @author Andres Ivan Medina
+ */
+
+@Composable
+fun OfficeView() {
+    val viewModel = OfficeViewModel()
+
+    LaunchedEffect(true) {
+        viewModel.getOfficeInformation()
+    }
+
+    val OfficeData = remember { viewModel.officeInfo }
```

Comment:
I think you don't need a remember here cause it's already a mutableStateOf and it lives inside of a view model

Reply:
Fixing it